src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -178,6 +178,20 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
     intelliJ.project.assertNoUncommittedChanges()
   }
 
+  @Test def mergeParentIntoCurrentBranch(): Unit = {
```

Comment:
Due to the overhead of each test case (spinning up a new IntelliJ instance), I'd rather merge this two cases into one... also, pls open a task to do the same with some of the existing test pairs for current/non-current branch

Reply:
https://github.com/VirtusLab/git-machete-intellij-plugin/issues/843